ts/Accessibility/Components/AnnotationsA11y.ts

```diff
@@ -13,7 +13,8 @@
 'use strict';
 
 import type Annotation from '../../Extensions/Annotations/Annotations';
-import type Point from '../../Core/Series/Point';
+import type AnnotationChart from '../../Extensions/Annotations/AnnotationChart';
+import type PointClass from '../../Core/Series/Point';
```

Comment:
As there is no name conflict, `import type Point` should be the favored pattern.

Reply:
Oi, and this needs to be done.